### ISSUE TITLE
[ListView] Unable to say when scrolling has stopped

### ISSUE THREAD

**Issue by NishanthShankar** [facebook/react-native#1832]
Unable to fire both "onScrollAnimationEnd" or "onMomentumScrollEnd" from ListView. Any workaround ?

**Comment by browniefed** [comment]
@NishanthShankar anything more on this? Did you figure it out?

**Comment by NishanthShankar** [comment]
Nope.. I haven't been able to detect small scrolls with precision. They are not very reliable yet
